desk/app/broadcaster.hoon

```diff
@@ -1,48 +1,59 @@
 ::  broadcaster: multi-target dms
 ::
-/-  c=chat
-/+  cj=channel-json,
+/-  c=chat, ch=channels, c3=chat-3
+/+  cj=channel-json, dm,
     dbug, verb
 ::
 |%
-+$  state-0
-  $:  %0
++$  versioned-state  $%(current-state state:v0)
++$  current-state
+  $:  %1
       cohorts=(map @t cohort)
   ==
-::
 +$  cohort
   $:  targets=(set ship)
       logging=(list relive)
       outward=(list writ:c)  ::NOTE  invented-here fake seal?
   ==
-::
 +$  relive
   $:  wen=@da
   $%  [%add targets=(set ship)]
       [%del targets=(set ship)]
-      [%msg =story:d:c]
+      [%msg =story:ch]
       [%err err=@t]
   ==  ==
-::
 +$  action
   $%  [%add-cohort cohort=@t targets=(set ship)]
       [%del-cohort cohort=@t targets=(set ship)]  ::  ~ for full deletion
-      [%broadcast cohort=@t =story:d:c]
+      [%broadcast cohort=@t =story:ch]
       [%delete cohort=@t time-id=@da]
   ==
-::
-++  update
++$  update
   $%  action
   ==
 ::
+++  v0
+  |%
+  +$  state
+    $:  %0
+        cohorts=(map @t cohort)
+    ==
+  ::
+  +$  cohort
+    $:  targets=(set ship)
+        logging=(list relive)
+        outward=(list writ:c3)  ::NOTE  invented-here fake seal?
+    ==
+  --
+::
```

Comment:
While we do use state version anchor for types, so this is much appreciated, the current practice is to keep the state types explicitly versioned with a tag. I think in general it makes it for easier reading, since all state types are usually aggregated in the same place. My suggestion here is to define `$state-0` outside of the `|v0` core.